src/charm.py

```diff
@@ -112,41 +112,40 @@ def __init__(self, *args):
         self.framework.observe(self._common_database.on.endpoints_changed, self._configure_webui)
         self.framework.observe(self._auth_database.on.endpoints_changed, self._configure_webui)
         self.framework.observe(
-            self.on.sdcore_management_relation_joined, self._publish_sdcore_management_url
+            self.on.sdcore_management_relation_joined, self._configure_webui
         )
         # Handling config changed event to publish the new url if the unit reboots and gets new IP
-        self.framework.observe(self.on.config_changed, self._publish_sdcore_management_url)
+        self.framework.observe(self.on.config_changed, self._configure_webui)
 
-    def _configure_webui(self, event: EventBase) -> None:
-        """Handle config changes.
+    def _configure_webui(self, _: EventBase) -> None:
+        """Configure Webui configuration file and publishes the Webui management URL.
```

Comment:
```suggestion
        """The main callback method for Juju events.
```

Reply:
> 

The first line of the docstring should be imperative. "The main callback method for Juju events." is not acceptable in format checking.

Reply:
The docstring is updated.